bin/cml-publish.js

```diff
@@ -52,12 +38,35 @@ const argv = yargs
     'gitlab-uploads',
     'Uses GitLab uploads instead of CML storage. Use GitLab uploads to get around CML size limitations for hosting artifacts persistently. Only available for GitLab CI.'
   )
+  .deprecateOption('gitlab-uploads', 'Use native instead')
+  .boolean('native')
+  .describe(
+    'native',
+    "Uses driver's native capabilities to upload assets instead of CML's backend."
```

Comment:
`backend` might be the best word - please see the comment above

Reply:
Im confused here... not sure if you are asking to change ```native``` to ```backend``` or you are pointing is not the best word... A few iterations before I called it ```backend```

Reply:
sorry, I mean "backend might be NOT the best word" (even in the text). and I suggested changing the description, not the option name.